lib/lib.nix

```
@@ -139,4 +179,4 @@
     {rawNickel = nickelResult;}
     // (importFromNickel flakeInputs system baseDir (builtins.fromJSON
         (builtins.unsafeDiscardStringContext (builtins.readFile nickelResult))));
-in {inherit importNcl;}
+in {inherit importNcl buildLockFile;}
```

Please put it on separate lines and also export `buildLockFileContents`.

```suggestion
in {
  inherit
    importNcl
    buildLockFile
    buildLockFileContents
    ;
}
```